apis/models/userDao.js

```diff
@@ -1,18 +1,43 @@
 const database = require('./dataSource');
 
-// const signUp = async () => {
-//     await database.query(
+const getUserByUsername = async (username) => {
+  const [user] = await database.query(
+    `
+    SELECT
+            id,
+            username,
+            fullname,
+            password,
+            phone_number,
+            address,
+            gender,
+            birth
+    FROM users
+    WHERE username = ?
+        `,
+    [username]
```

Comment:
줄바꿈으로 인한 들여씌기 할 때 2칸 기준을 잘 맞춰주면 가독성에 더 좋습니다.
```suggestion
    SELECT
        id,
        username,
        fullname,
        password,
        phone_number,
        address,
        gender,
        birth
    FROM users
    WHERE username = ?
    `,
    [username]
```

Reply:
수정